ATOM/Dependencies/Detectron/Optimizations/Disable startups.ps1

```diff
@@ -40,36 +41,43 @@ $startups = @(
 	"GogGalaxy",
 	"HPOneAgentService",
 	"HPSEU_Host_Launcher",
+	"IntelGraphicsAGS",
+	"iTunesHelper",
 	"Lync",
+	"Opera Browser Assistant",
 	"Overwolf",
 	"Parsec.App.0",
+	"Skype for Desktop",
 	"Spotify",
 	"SunJavaUpdateSched",
 	"Steam",
 	"TeamsMachineInstaller"
 )
 
-foreach ($startupPath in $startupPaths) {
-	$startupKeys = Get-ItemProperty -Path $startupPath -ErrorAction SilentlyContinue
-	foreach ($startup in $startups) {
-		$startupValue = $startupKeys.$startup
-		if ($startupValue -ne $null) {
-			for ($i = 0; $i -lt 2; $i++) {
-				if ((Compare-Object $startupValue $startupEnabled[$i] -SyncWindow 12) -eq $null) {
-					Set-ItemProperty -Path $startupPath -Name $startup -Type Binary -Value $startupDisabled[$i]
-					Write-OutputBox "- $startup has been disabled."
-					break
+	foreach ($startupPath in $startupPaths)	{
+		$startupKeys = [Microsoft.Win32.Registry]::GetValue($startupPath, $null, $null)

```

Comment:
I'd prefer to stick to PowerShell native commands since they're more readable and better documented. 2% performance improvement is in margin of error and likely won't noticeably improve anything.

Reply:
Sure, will roll back two commits.